app/collins/models/Asset.scala

```diff
@@ -266,7 +266,7 @@ object Asset extends Schema with AnormAdapter[Asset] with AssetKeys {
     val findLocations = find(PageParams(0, 50, "ASC", "tag"), AttributeResolver.emptyResultTuple, instanceFinder)
       .items
       .collect { case a: Asset => a }
-      .filter { _.tag != MultiCollinsConfig.thisInstance }
+      .filter(asset => { !MultiCollinsConfig.thisInstance.contains(asset.tag) })
```

Comment:
This is just for finding the datacenter assets. Before actually executing the query the user sent, we have to figure out which remote collins instances are around that we need to send the query to. 

What we actually do here is query for assets of type `MultiCollinsConfig.instanceAssetType` which is the asset type that represents instances in other datacenters. This filter will remove one asset, and that is the local datacenter.